src/components/Header/Header.tsx

```diff
@@ -0,0 +1,26 @@
+import type { FC, HTMLProps, ReactNode } from 'react';
+import variables from '../../styles/scss/variables.module.scss';
+
+import logo from '../../assets/img/logo-icon.png';
+
+import './Header.scss';
+
+const { prefix } = variables;
+
+export interface HeaderProps extends HTMLProps<HTMLDivElement> {
+  logoUrl?: string;
+  children?: ReactNode;
+}
+
+export const Header: FC<HeaderProps> = ({
+  logoUrl,
+  children,
+  ...otherProps
+}) => (
+  <header className={`${prefix}-header`} {...otherProps}>
+    <a id="logo" href={logoUrl || '/'}>
+      <img src={logo} alt="Scrabble logo" />
```

Comment:
Will it be ok to have a fixed logo img for a reusable component library?

Reply:
Very good point - this should be made as the default logo but a way to provide a custom one should be provided